configuration/enable-alert-data-retention.adoc

```diff
@@ -0,0 +1,30 @@
+[id="enable-alert-data-retention"]
+= Enabling alert data retention
+include::modules/common-attributes.adoc[]
+:context: enable-alert-data-retention
+
+toc::[]
+
+[role="_abstract"]
+Learn how to configure a retention period for {product-title}'s alerts.
```

Comment:
I think we usually avoid adding "'s" to things like product names. I think you can remove it and have the meaning be the same.